app/views/shared/_authentication_title.html.erb

```diff
@@ -0,0 +1 @@
+<a href="/"><%= community_name %></a> is a community of <%= display_estimated_user_count? ? number_with_delimiter(estimated_user_count) : "" %> amazing <%= community_members_label %>
```

Comment:
Just wondering... Should we start using  `<%= root_path %>` instead of `/` in href attribute?